Cargo.toml

```diff
@@ -9,6 +9,144 @@ members = [
 ]
 resolver = "2"
 
+[workspace.dependencies]
+log = { version = "0.4.17", default-features = false }
+env_logger = "0.10.0"
+twox-hash = { version = "1.6.3", default-features = false, features = ["digest_0_10"]}
+thiserror = "1.0.40"
+apache-avro = { version = "0.14.0", default-features = false }
+
+# substrate wasm
+codec = { package = "parity-scale-codec", version = "3.6.1", default-features = false  }
+parity-scale-codec = { version = "3.6.1", default-features = false }
```

Comment:
Here `codec === parity-scale-codec`.
Can we use one or the other of these aliases throughout our code? i.e. rename any existing `parity-scale-codec` references to `codec` and then delete the `parity-scale-codec` reference.

[The Cargo Book-Renaming dependencies](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#renaming-dependencies-in-cargotoml)